Source/WebKit/WebProcess/GPU/media/MediaPlayerPrivateRemote.cpp

```diff
@@ -423,7 +424,7 @@ void MediaPlayerPrivateRemote::currentTimeChanged(const MediaTime& mediaTime, co
     m_cachedMediaTime = mediaTime;
     m_cachedMediaTimeQueryTime = queryTime;
 
-    if (reverseJump) {
+    if (reverseJump || (!oldCachedTime && oldCachedTime != mediaTime)) {
```

Comment:
Should it be  `(oldCachedTime && oldCachedTime != mediaTime)`?

Reply:
no, it needs to be done only when the previous time was 0 (eg. playback didn't start)

I'm actually guessing it should be done if we go from `m_timeIsProgressing = false` to `m_timeIsProgressing = true` but as this code has caused lots of regression in the past, I prefer to take the safe approach and narrow the scope

Reply:
Then let's write it as: `(!oldCachedTime && !!mediaTime)`?